dev-docs/RFCs/v7.0/google-maps-integration.md

```diff
@@ -1,6 +1,6 @@
 # RFC: Google Maps Integration
 
-* **Authors**: Xiaoji Chen
+* **Authors**: Xiaoji Chen, Don MacCurdy
```

Comment:
While "Mac" and "Mc" are [interchangeable in library sorting and voter registration](https://en.wikipedia.org/wiki/Mac_and_Mc_together), "Mc" would be preferable. 😄

Reply:
🤦‍♂️